src/components/MessageList/MessageList.tsx

```diff
@@ -880,7 +872,13 @@ const MessageListWithContext = <
         inverted={inverted}
         keyboardShouldPersistTaps='handled'
         keyExtractor={keyExtractor}
-        ListEmptyComponent={renderEmptyStateIndicator}
+        ListEmptyComponent={
+          <View style={styles.flex}>
+            <View style={styles.flex} testID='empty-state'>
+              <EmptyStateIndicator listType='message' />
+            </View>
+          </View>
+        }
```

Comment:
Fixes that wierd inverted empty list component issue with FlatList.

Reply:
wait what is this fixing and why would moving it like this fix it? anonymous functions aren't recommended https://reactnative.dev/docs/optimizing-flatlist-configuration#avoid-anonymous-function-on-renderitem

Reply:
There is this long running issue with RN - https://github.com/facebook/react-native/issues/21196

Let me just add `scaleY: -1` to it instead then, and keep this out of anonymous function

Reply:
Actually, just saw that we don't render anything in EmptyStateIndicator by default. So we don't need to fix anything OOTB. Removed anonymous function!!